src/blocks/cards.scss

```diff
@@ -0,0 +1,117 @@
+:root {
+  --grey-color: #616070;
+  --button-color: #00acdc;
+  --button-second-color: #fff;
+  --color-title-rev-price: #060b35;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  color: var(--color-title-rev-price);
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+}
+
+.card__container {
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  margin-bottom: 40px;
+  align-self: center;
+}
```

Comment:
Ohhhh....you need to rewrite all your `.scss` files....
![opera_2RXR04N6LS](https://user-images.githubusercontent.com/104684446/185989565-077cf62d-a3ef-491c-95c9-dcc64735aa42.png)
.